steps/PageSlugs.ts

```diff
@@ -30,6 +30,7 @@ enum PageSlugs {
   AboutProperty = "about-property",
   PropertyOccupation = "property-occupation",
   Gas = "gas",
+  DecorationPack = "Decoration pack",
```

Comment:
I think you also need to add a reference to the step in the `slugs` object in this file